ee-feature-pack/common/src/main/resources/modules/system/layers/base/org/hibernate/main/module.xml

```diff
@@ -56,5 +56,7 @@
         <module name="net.bytebuddy"/>
         <module name="java.xml"/>
         <module name="com.h2database.h2" optional="true"/>
+        <module name="com.oracle.ojdbc" optional="true"/>
```

Comment:
A comment here would be good, e.g.

```
<!-- If the Oracle JDBC driver classes are accessible, Hibernate will use them to tailor OracleDialect behavior -->
```

I assume that statement is accurate; if not we should adapt it. It's deliberately vague to help avoid it falling out of date.

Reply:
:+1: for leaving a comment there as that also gives a hint to future maintainers as to why it is there.

Reply:
Thanks. And yes, maintainability is the main purpose I had in mind. Any optional module dependency begs the question of why it's optional and what conditions would make it required.